src/shims/ApiCompat.proj

```diff
@@ -40,6 +41,16 @@
     </Exec>
 
     <Exec Command="$(ApiCompatCmd) &quot;$(NetStandardRefPath)&quot; @&quot;$(ApiCompatResponseFile)&quot; &gt; $(ApiCompatNSBaselineFile)"
+          Condition="'$(TargetGroup)' != 'netfx'"
```

Comment:
This are the valid configurations for this project: https://github.com/dotnet/corefx/blob/master/src/shims/dir.props#L4-L8